### PR TITLE
Add link to OmniSharp-vscode debugger page

### PR DESCRIPTION
**Pull request by kendrahavens**
Add link to OmniSharp-vscode debugger page

## Suggested Reviewers

@mairaw

### CODE REVIEW

File: docs/csharp/getting-started/with-visual-studio-code.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
 7. Hit the green arrow at the top to continue debugging or press the red square to stop.
 
+> TIP: For a more in-depth description and troubleshooting tips on .NET Core debugging with OmniSharp in VS Code visit the [OmniSharp-VSCode repository on GitHub](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md)
```
Comment (mairaw) [inline docs/csharp/getting-started/with-visual-studio-code.md]: Suggested rewriting:

```markdown
> [TIP]
For more information and troubleshooting tips on .NET Core debugging with OmniSharp in VS Code, see [Instructions for setting up the .NET Core debugger](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md).
```

**Review by mairaw — COMMENTED** [review]
I think we should also link to http://www.omnisharp.net/ on the first mention of Omnisharp in the article. Newbies might not know what Omnisharp is and might want to get more info,

File: docs/csharp/getting-started/with-visual-studio-code.md [hunk 2]
```diff
@@ -89,6 +89,8 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
 7. Hit the green arrow at the top to continue debugging or press the red square to stop.
 
+> TIP: For more information and troubleshooting tips on .NET Core debugging with OmniSharp in VS Code, see [Instructions for setting up the .NET Core debugger](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md).
```
Comment (svick) [inline docs/csharp/getting-started/with-visual-studio-code.md]: Shouldn't this use the `> [!TIP]` syntax?

Comment (mairaw) [inline docs/csharp/getting-started/with-visual-studio-code.md]: Yes! For the previous one we can't because of a bug.

**Comment by BillWagner** [comment]
closing and reopening to force a new OPS build.

**Comment by BillWagner** [comment]
LGTM.  Let's :shipit:

**Pull request merged**